rawterm-1.1.0-2.rockspec

```diff
@@ -1,5 +1,5 @@
 package = "rawterm"
-version = "1.1.0-1"
+version = "1.1.0-2"
```

Comment:
Shouldn't this be 1.2.0-1? It's a new feature, not just a rockspec change..

Reply:
👌

Reply:
I've changed the version to `1.2.0-1`